zephyr/Makefile

```diff
@@ -71,15 +70,10 @@ $(GENERIC_TARGETS): $(LIBMICROPYTHON)
 $(CLEAN_TARGETS):  clean
 
 $(GENERIC_TARGETS) $(KCONFIG_TARGETS) $(CLEAN_TARGETS):
-	$(RM) -f outdir/$(OUTDIR_PREFIX)/zephyr.lnk
```

Comment:
I did but since what I wrote was hopelessly broken Amit pulled it just before v1.6 went out of the door (I did have a fix but is was pretty ugly).